demo/api/src/db/fixtures/generators/blocks/page-content.fixture.ts

```diff
@@ -0,0 +1,83 @@
+
+@Injectable()
+export class PageContentBlockFixtureService {
+    constructor(
+        private readonly richtextGeneratorService: RichTextBlockFixtureService,
+        private readonly damImageBlockGenerator: DamImageBlockFixtureService,
+        private readonly textImageBlockGenerator: TextImageBlockFixtureService,
+        private readonly spaceBlockGenerator: SpaceBlockFixtureService,
+        private readonly headlineBlockGenerator: HeadlineBlockFixtureService,
+        private readonly anchorBlockGenerator: AnchorBlockFixtureService,
+        private readonly fullWithBlockGenerator: FullWidthImageBlockFixtureService,
+        private readonly linkListBlockGenerator: LinkListBlockFixtureService,
+        private readonly youTubeVideoBlockGenerator: YouTubeVideoBlockFixtureService,
+        private readonly damVideoBlockGenerator: DamVideoBlockFixtureService,
+        private readonly columnsBlockGenerator: ColumnsBlockFixtureService,
+        private readonly mediaBlockGenerator: MediaBlockFixtureService,
+        private readonly videoBlockGenerator: VideoBlockFixtureService,
+        private readonly twoListsBlockGenerator: TwoListsBlockFixtureService,
+    ) {}
+
+    async generateBlock(): Promise<ExtractBlockInput<typeof PageContentBlock>> {
+        const blocks: ExtractBlockInputFactoryProps<typeof PageContentBlock>["blocks"] = [];
+
+        const blockCfg: Record<keyof typeof supportedBlocks, BlockFixture> = {
+            richtext: this.richtextGeneratorService,
+            image: this.damImageBlockGenerator,
+            textImage: this.textImageBlockGenerator,
+            space: this.spaceBlockGenerator,
+            linkList: this.linkListBlockGenerator,
+            fullWidthImage: this.fullWithBlockGenerator,
+            anchor: this.anchorBlockGenerator,
+            columns: this.columnsBlockGenerator,
+            damVideo: this.damVideoBlockGenerator,
+            headline: this.headlineBlockGenerator,
+            youTubeVideo: this.youTubeVideoBlockGenerator,
+            media: this.mediaBlockGenerator,
+            video: this.videoBlockGenerator,
+            twoLists: this.twoListsBlockGenerator,
+        };
+
+        for (let i = 0; i < 10; i++) {
+            const [type, generator] = random.arrayElement(Object.entries(blockCfg));
+            if (generator) {
+                const props = await generator.generateBlock();
+                const userGroup = random.arrayElement(Object.values(UserGroup));
+                blocks.push({
+                    key: String(i),
+                    visible: datatype.boolean(),
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    type: type as any,
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    props: props as any,
+                    // @ts-expect-error wrong typing?
```

Comment:
Should be fixed in main

Reply:
Unfortunately it is not fixed in the main branch, the only difference is, that in the main branch the userGroup value is not specified here: 
https://github.com/vivid-planet/comet/blob/main/demo/api/src/db/fixtures/generators/blocks/blocks.generator.ts#L32

Reply:
Okay, we should investigate this in a follow-up PR. This ts-expect-error shouldn't be necessary in applications